pom.xml

```diff
@@ -139,7 +139,7 @@ flexible messaging model and an intuitive client API.</description>
     <log4j2.version>2.18.0</log4j2.version>
     <bouncycastle.version>1.69</bouncycastle.version>
     <bouncycastlefips.version>1.0.2</bouncycastlefips.version>
-    <jackson.version>2.13.4</jackson.version>
+    <jackson.version>2.13.4.20221013</jackson.version>
```

Comment:
2.13.4.2?

Reply:
This property indicates the `bom` 
https://mvnrepository.com/artifact/com.fasterxml.jackson/jackson-bom/2.13.4.20221013